common/core/hwclayer.cpp

```diff
@@ -162,11 +163,17 @@ void HwcLayer::SetVisibleRegion(const HwcRegion& visible_region) {
 
 void HwcLayer::SetReleaseFence(int32_t fd) {
   if (release_fd_ > 0) {
-    close(release_fd_);
-    release_fd_ = -1;
-  }
-
-  release_fd_ = fd;
+    if (fd != -1) {
```

Comment:
hmm can u explain more. This is supposed to replace the current fence with a new fence.

Reply:
@kalyankondapally 
-1 means to reset the fence
otherwise accumulate

Reply:
@kalyankondapally 
When masaic mode or clone mode, the kms_fence should be accumulate to hwc_layer layer release fence.
Otherwise the release fence is not accurate, will cause potencial errors